build/setup.d/10-upgrade-base.sh

```diff
@@ -6,6 +6,6 @@ echo "Updating system..."
 
 sudo rm -rf /var/lib/apt/lists/*
 apt-get update -y  # -q >>/tmp/build/upgrade.log
-apt-mark hold openssh-server
-apt-get dist-upgrade -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" -y # -q  >>/tmp/build/upgrade.log
-aptitude unhold openssh-server
+#apt-mark hold openssh-server
+#apt-get dist-upgrade -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" -y # -q  >>/tmp/build/upgrade.log
```

Comment:
We'll get this fixed :+1: Maybe by using the most recent Ubuntu AMI via script on AWS.